icubam/messaging/sms_sender_test.py

```diff
@@ -21,6 +21,13 @@ def test_from_string(self):
       sms_sender.get(self.config, 'Fake'), sms_sender.FakeSender)
     self.assertIsInstance(
       sms_sender.get(self.config, 'fake'), sms_sender.FakeSender)
+    with self.assertRaises(ValueError):
+      sms_sender.get(self.config, 'dummy_sms_sender_name')
+
 
   def test_from_config(self):
     self.assertIsInstance(sms_sender.get(self.config), sms_sender.FakeSender)
+
+  def test_send_from_fake_sender(self):
+    ms = sms_sender.get(self.config, 'Fake')
+    ms.send("Fake phone number", "Hello from tests")
```

Comment:
Maybe just add an inline comment explaining what this does: i.e. I hope this actually won't send a SMS if the token is provided..

Reply:
Fake sender is safe.

Reply:
I add a comment to make it clearer